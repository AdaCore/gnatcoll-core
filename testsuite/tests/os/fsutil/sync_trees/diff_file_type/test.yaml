description: Test for GNATCOLL.OS.FSUtil.Sync_Trees. Scenario with different
             files type
driver: build_run_assert_and_diff
data:
  - "src_top_dir"
  - "dst_top_dir"
source_dirs:
  - "../"