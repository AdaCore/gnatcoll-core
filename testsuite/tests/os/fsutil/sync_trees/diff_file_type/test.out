Test for GNATCOLL.OS.FSUtil.Sync_Trees
OK   test.adb:15
Source tree
===========
FILE src_top_dir/0
FILE src_top_dir/1
FILE src_top_dir/2

Destination tree
================
FILE dst_top_dir/0
FILE dst_top_dir/1
FILE dst_top_dir/2

Content checking
================
<=== TEST PASSED ===>
